_posts/2024-09-30-this-month-in-servo.md

```diff
@@ -0,0 +1,510 @@
+    1346643727336485b8ee19b3db6d4a28366e89f1	https://github.com/servo/servo/pull/33549	Assert that we don't get malformed ContentSizes in tables (#33549)
+    7fdaccde5501283708503e9a64efa578ac9ae7f7	https://github.com/servo/servo/pull/33512	ohos: Support product flavors (#33512)
+-->
+
+Servo has had several new features land in our nightly builds over the last month:
+
+- as of 2024-09-10, we now support **&lt;ul type>** and **&lt;ul compact>** (@simonwuelker, #33303)
+- as of 2024-09-10, we now support **console.timeLog()** (@simonwuelker, #33377)
+- as of 2024-09-10, we now support the **encodeInto() method on TextEncoder** (@webbeef, #33360)
+- as of 2024-09-10, we now support **&lt;link rel=prefetch>** (@simonwuelker, #33345)
+- as of 2024-09-12, we now support **right-to-left languages**, except for floats (@mrobinson, @atbrakhi, #33375)
+- as of 2024-09-14, we now support **‘table-layout: fixed’** (@Loirooriol, #33384, #33442)
+- as of 2024-09-17, we now support the **‘reset’ event on XRReferenceSpace** properties (@mrobinson, @Loirooriol, #33479)
+- as of 2024-09-19, we now support the **‘object-fit’** and **‘object-position’** properties (@mrobinson, @Loirooriol, #33479)
+- as of 2024-09-19, **Crypto.getRandomValues()** can now take **BigInt64Array** or **BigUint64Array** (@msub2, #33485)
+- as of 2024-09-25, we now support **‘innerText’** and **‘outerText’ on HTMLElement** (@Melchizedek6809, #33312)
+
+Servo’s flexbox support continues to mature, with support for **‘align-self: normal’** (@Loirooriol, #33314), plus corrections to **cross-axis percent units** in descendants (@Loirooriol, #33242), **automatic minimum sizes** (@Loirooriol, #33248, #33256), **replaced flex items** (@Loirooriol, #33263), **baseline alignment** (@mrobinson, @Loirooriol, #33347), and **absolute descendants** (@mrobinson, @Loirooriol, #33346).
+
+Our table layout has improved, with support for **width** and **height presentational attributes** (@Loirooriol, #33405, #33425), as well as better handling of **‘border-collapse’** (@Loirooriol, #33452) and **extra &lt;col> and &lt;colgroup> columns** (@Loirooriol, #33451).
+
+We’ve also started working on the intrinsic sizing keywords **‘min-content’**, **‘max-content’**, **‘fit-content’**, and **‘stretch’** (@Loirooriol, #33492).
+Before we can support them, though, we needed to land patches to calculate intrinsic sizes, including for **percent units** (@Loirooriol, #33204), **aspect ratios** of replaced elements (@Loirooriol, #33240), **column flex containers** (@Loirooriol, #33299), and **‘white-space’** (@Loirooriol, #33343).
+
+We’ve heavily reworked our **WebGPU support**, with **faster canvas presentation** (@sagudev, #33387), support for **resizing the canvas** (@sagudev, #33521) and the **‘constants’ property on GPUProgrammableStage** (@sagudev, #33291), plus improvements to **GPUBuffer** (@sagudev, #33154) and the **destroy() method on GPUTexture** (@sagudev, #33534).
```

Comment:
```suggestion
We’ve also worked on our **WebGPU support**, with **faster canvas presentation** (@sagudev, #33387), **support pipeline-overridable constants** (@sagudev, #33291), completely reworked **GPUBuffer** to match the spec (@sagudev, #33154) and heavily reworked webgpu presentation to match the spec and properly support (re)configuration and resize on **GPUCanvasContext** (@sagudev, #33521).
```

Reply:
thanks! done in d4ed0826d05fd665103d956ac11add4ae5d487c6